release/windows/build_backend.ps1

```diff
@@ -1,9 +1,6 @@
 Remove-Item -Recurse -Force -ErrorAction SilentlyContinue ./build
 Remove-Item -Recurse -Force -ErrorAction SilentlyContinue ./dist
 
-pip install git+https://github.com/huggingface/huggingface_hub.git
```

Comment:
just wondering, we don't need the `huggingface_hub` dependenciy at all here?

Reply:
our DL models needs huggingface `transformers` and it needs `huggingface_hub`. We are going to remove the dependencies, see https://github.com/MannLabs/alphapeptdeep/issues/178

Reply:
The dependency is still Importen but just like all other dependencies. This Statement was included to get a Bugfix early which resolved some Import magic